lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -270,6 +270,17 @@
             - Name of the storage domain this virtual machine lease reside on.
             - NOTE - Supported since oVirt 4.1.
         version_added: "2.4"
+    host_devices:
+        description:
+            - Single Root I/O Virtualization - technology that allows single device to expose multiple endpoints that can be passed to VMs
```

Comment:
In this documentation we need to describe the feature a bit not so much detailed.
The first line is enough, the most important part is to define the structure of this variable, that it's list and it takes dictinary as a items and inside dictinary you can define `name` which is name of the device and the `state` where user can specify if it should be attached or detach of the VM.